app/views/admin/tracks/index.html.haml

```diff
@@ -53,6 +61,21 @@
               - else
                 %i.fa.fa-check
             %td
+              - if track.self_organized? && track.room
```

Comment:
Room is not mandatory for regular tracks as we know them till now, but can't we just be showing this? If and when an organizer might want to assign a room at a track, the organizer should be free to do that. Same with start/end dates. What do you think?